File: 3194-FindWordsContainingCharacter/3194-FindWordsContainingCharacter.py
# Last updated: 01/09/2025, 00:58:25
class Solution:
    def findWordsContaining(self, words: List[str], x: str) -> List[int]:
        ls = []
        k = 0
        for s in words:
            if x in s:
                ls.append(k)
            k += 1
        return ls

File: 3194-FindWordsContainingCharacter/3194-FindWordsContainingCharacter.cpp
// Last updated: 01/09/2025, 00:58:28
class Solution {
public:
    vector<int> findWordsContaining(vector<string>& words, char x) {
        vector<int>num;
        for(int i=0;i<words.size();i++){
            for(int j=0;j<words[i].size();j++){
                if(words[i][j]==x){
                    num.push_back(i); 
                    break;
                } 
            }
        }
        return num;
    }
};